geopy/distance.py

```diff
@@ -15,13 +15,29 @@
 :const:`distance.EARTH_RADIUS`, so it can be customized (it should
 always be in kilometers, however).
 
-Vincenty distance (:class:`.vincenty`) uses a more accurate ellipsoidal model
-of the earth. This is the default distance formula, and is thus aliased as
-``distance.distance``. There are multiple popular ellipsoidal models, and
-which one will be the most accurate depends on where your points are located
-on the earth. The default is the WGS-84 ellipsoid, which is the most globally
-accurate. geopy includes a few other
-models in the distance.ELLIPSOIDS dictionary::
+The geodesic distance is the shortest distance on the surface of an
+ellipsoidal model of the earth.  There are two algorithms used to
+compute this distance
+[Vincenty's method](https://en.wikipedia.org/wiki/Vincenty's_formulae)
+(:class:`.vincenty`) which is accurate to 0.2 mm and the distance
+calculation fails to converge for nearly antipodal points.
+
+If either of the python packages pyproj or geographiclib is available,
+then the method given by
+[Karney (2013)](https://doi.org/10.1007%2Fs00190-012-0578-z)
+(:class:`.geodesic`) is also provided.  This is accurate to round-off
+and always converges.  You can install pyproj and geographiclib with
+
+    pip install pyproj geographiclib
```

Comment:
Is there a reason to need to have both installed? From what I see here either one of them should be enough.